File: src/beginner_tutorials/src/move_turtle.cpp
#include "ros/ros.h"
#include "geometry_msgs/Twist.h"

int main(int argc, char **argv){
    ros::init(argc, argv, "turtle_mover");
    ros::NodeHandle n;
    ros::Publisher publisher=n.advertise<geometry_msgs::Twist>("/turtle1/cmd_vel",1000);
    ros::Rate loop_rate(2);
    while (ros::ok()){
        geometry_msgs::Twist twist;
        twist.linear.x = 1.0;
        twist.angular.z = 1.5;
        publisher.publish(twist);
        ros::spinOnce();
        loop_rate.sleep();
    }
    return 0;
}

File: src/beginner_tutorials/src/scripts/move_turtle.py
#!/usr/bin/env python
import rospy
from geometry_msgs.msg import Twist
from time import sleep

def turtle_mover():
    publisher = rospy.Publisher("/turtle1/cmd_vel", Twist, queue_size=10)
    rospy.init_node("turtle_mover")
    twist = Twist()
    while not rospy.is_shutdown():
        twist.linear.x  = 1.0
        twist.angular.z = 1.5
        rospy.loginfo("Moving turtle:\n%s"%(twist))
        publisher.publish(twist)
        sleep(.5)

if __name__ == "__main__":
    try:
        turtle_mover()
    except rospy.ROSInterruptException:
        pass
